docs/go-run-your-first-app.md

```diff
@@ -99,13 +116,34 @@ Task Queues are defined by a simple string name:
 <!--SNIPSTART money-transfer-project-template-go-shared-task-queue-->
 <!--SNIPEND-->
 
+### Running the Worker
+
 Run worker/main.go from the project root using the following command:
 
-```
+```bash
 go run worker/main.go
 ```
 
-When you start the Worker it begins polling the Task Queue. The first Task the Worker finds is the one that tells it to execute the Workflow function. The Worker communicates the event back to the server which then causes the server to send Activity Tasks to the Task Queue as well. The Worker then grabs each of the Activity Tasks in their respective order from the Task Queue and executes each of the corresponding Activities.
+When you start the Worker it begins polling the Task Queue (if you check Temporal Web again, you will see a new Poller registered where previously there was none). 
+
+The terminal output will look like this:
+
+```bash
+2021/03/12 01:43:49 INFO  No logger configured for temporal client. Created default one.
+2021/03/12 01:43:49 INFO  Started Worker Namespace default TaskQueue TRANSFER_MONEY_TASK_QUEUE WorkerID 41326@swyxs-Mac-mini@
+2021/03/12 01:43:50 DEBUG ExecuteActivity Namespace default TaskQueue TRANSFER_MONEY_TASK_QUEUE WorkerID 41326@swyxs-Mac-mini@ WorkflowType TransferMoney WorkflowID transfer-money-workflow RunID 0730a9a3-d17b-4cb5-a4a0-279c9759dfa1 ActivityID 5 ActivityType Withdraw
+
+Withdrawing $54.990002 from account 001-001. ReferenceId: 8e37aafe-5fb8-4649-99e3-3699e35e6c32
+```
+
+> Tip: What actually happens under the hood:
```

Comment:
This might be a good spot for a info/tip block:

```suggestion
:::tip What actually happens under the hood
```
https://v2.docusaurus.io/docs/markdown-features/admonitions

Reply:
Otherwise, 
```suggestion
**Tip**

 What actually happens under the hood:
```